src/libsyntax/feature_gate.rs

```diff
@@ -383,6 +383,9 @@ declare_features! (
 
     // allow '|' at beginning of match arms (RFC 1925)
     (active, match_beginning_vert, "1.21.0", Some(44101)),
+
+    // allow `'_` placeholder lifetimes
+    (active, underscore_lifetimes, "1.21.0", Some(44524)),
```

Comment:
Should be 1.22.0? (BTW `match_beginning_vert` should also be 1.22.0?)

Reply:
I never know what to actually put here.